### PR TITLE
[FMR]:Show current and new version on update dialog

### PR DESCRIPTION
**Pull request by SinghRajenM**
Fixes issue #19

### param -px64 -v1.234
image

### param -px64 -v8.19
image

### CODE REVIEW

File: src/winmain.cpp [hunk 1]
```diff
@@ -1311,8 +1352,21 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR lpszCmdLine, int)
...
+		if (versionCurrent.empty())
+			versionCurrent = MSGID_VERSIONCURRENT;
+		versionCurrent += L" " + productVersionToFileVersion(version);
+
+		wstring versionNew = nativeLang.getMessageString("MSGID_VERSIONNEW");
+		if (versionNew.empty())
+			versionNew = MSGID_VERSIONNEW;
+		versionNew += L" " + gupDlInfo.getVersion();
+
+		updateAvailable += L"\n\n" + versionCurrent;
```
Comment (SinghRajenM) [inline src/winmain.cpp:1368]: Npp `gup.xml` is set to `<MessageBoxTitle isModal="no">`. Hence this change is not applied if `isModel` is set to `yes`. However, that modal dialog is not updated for long I guess, as I could see the translation is also not updated. 
Pls see here (line 1325)- 
https://github.com/notepad-plus-plus/wingup/blob/50021b7411ea3204bd2564cbfc168601d6870b09/src/winmain.cpp#L1320-L1327

**Comment by donho** [comment]
@SinghRajenM 
Sorry I didn't see your PR - thanks to Chinese Spam attacks I just noticed this useful feature!

File: src/winmain.cpp (hunk 1 above)
Comment (donho) [reply src/winmain.cpp:1368]: Thank you for your heads up.
I'll make it work.

**Comment by SinghRajenM** [comment]
> @SinghRajenM
> Sorry I didn't see your PR - thanks to Chinese Spam attacks I just noticed this useful feature!

@donho 
No worries. Sometime things take time.

**Pull request closed without merging**